src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcome.cshtml

```diff
@@ -197,10 +199,31 @@
                                           id="OptionFailedText"
                                           name="OptionFailedText"
                                           rows="5"
-                                          aria-describedby="OptionFailedText-info">@Model.OptionFailedText</textarea>
+                                          aria-describedby="OptionFailedText-info OptionFailedText--hint">@Model.OptionFailedText</textarea>
                                 <span id="OptionFailedText-info" class="govuk-hint govuk-character-count__message" aria-live="polite"></span>
                             </div>
                         </div>
+                        <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="500">
```

Comment:
Probably should be 150 words here

Reply:
That's Fail - External comments right? AC3 and the content doc both say 500 words.